mockserver/server.js

```diff
@@ -45,8 +45,14 @@ app.get('/v8/gfs/solar', (req, res, next) => {
   });
 });
 
-app.get('/feature-flags', (req, res) => {
-  res.json({});
+app.get('/v8/meta', (req, res) => {
+  res.json({
+    features: {
+      'feature-flag-1': true,
+      'flag-2': false,
+    },
+    callerLocation: [0, 0],
```

Comment:
```suggestion
    callerLocation: [138, 36],
```
This should help it pass the new cypress test